index.js

```diff
@@ -37,7 +38,11 @@ module.exports = {
     global.EmberENV = EmberENV;
 
     var Compiler = require(templateCompilerPath);
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile); // jshint ignore:line
+    var templateCompilerFullPath = require.resolve(templateCompilerPath);
+    var templateCompilerCacheKey = fs.readFileSync(templateCompilerFullPath, { encoding: 'utf-8' });
```

Comment:
i dont think this is sufficient, the templateCompiler may change on each rebuild, this merely checks the cache on start.